### ISSUE TITLE
The network details for this chain ID do not match our records. 

### ISSUE THREAD

**Issue by DaRealX** [omgnetwork/optimism-v2#226]
Hi team.

I removed the Boba L2 network from Metamask.
Then I went back to the Gateway, I click connect using browser, brings me to the wallet but when I click airdrop,
I get the above message (see attached screendump).

Kind regards,

Lex

![Metamask_Warning_Boba_Network_Details_Are_Different_12-12-2021](https://user-images.githubusercontent.com/80894290/145712367-c8d84564-2e50-4df9-88a2-4bc1a37fc1e6.png)

**Comment by CAPtheorem** [comment]
Yes, that is the default behavior. Check the values: `mainnet.boba.network` and `Chain ID = 288`, and if they are correct, proceed.

**Comment by DaRealX** [comment]
OK just to make sure I understand it correctly. 
Regardless if the info (Name, Network, Chain ID) is correct or not, that warning on the bottom will always be shown?
They could just say: Please verify network details before proceeding.
